### PR TITLE
Remove outdated warnings about IPVS

### PR DESCRIPTION
**Pull request by champtar**

### CODE REVIEW

File: website/content/installation/network-addons.md [hunk 1]
```diff
@@ -25,11 +25,3 @@ Kube-router   | Mostly (see [known issues]({{% relref "configuration/kube-router
...
 Weave Net     | Mostly (see [known issues]({{% relref "configuration/weave.md" %}}))
 
-### IPVS mode in kube-proxy
-
-Starting in Kubernetes 1.9, `kube-proxy` has beta support for a more
-efficient "IPVS mode", in addition to the default "iptables mode."
-MetalLB **might** work with IPVS mode in kube-proxy, in Kubernetes
-1.13 or later. However, it is not explicitly tested yet, so it's at
-your own risk. See our [tracking
-bug](https://github.com/metallb/metallb/issues/153) for details.
```
Comment (russellb) [inline website/content/installation/network-addons.md:35]: This issue isn't closed, though.  Maybe it should be?  I'd like to close this issue before merging this.

**Review by russellb — CHANGES_REQUESTED** [review]

File: website/content/installation/network-addons.md (hunk 1 above)
Comment (champtar) [reply website/content/installation/network-addons.md:35]: How to use MetalLB with IPVS is properly documented, it's working, so yes for we can close #153 IMO

Comment (russellb) [reply website/content/installation/network-addons.md:35]: closed

**Comment by russellb** [comment]
also cherry-picked this to v0.9 and live-website branches

**Pull request merged**